### DISCUSSION TITLE
抽屉有没有打开关闭的绑定事件?

### DISCUSSION THREAD

**Discussion by mimimiaomiao1** [HandyOrg/HandyControl#1334] [Q&A] [answered]
<img width="441" alt="image" src="https://user-images.githubusercontent.com/58240619/211503708-471889fc-ca1e-4136-80a0-8f8b90e9427e.png">

viewmodel,如何来打开关闭抽屉.

**Comment by NaBian** [comment, marked as answer]
```IsOpen``` ```Opened``` ```Closed```
